packages/eslint-config/index.js

```diff
@@ -1,13 +1,22 @@
+const { getPackageInfoSync } = require('local-pkg')
+
+const nuxt = getPackageInfoSync('nuxt')
+const isNuxt2 = nuxt && nuxt.version && nuxt.version.startsWith('2.')
+// console.log({ isNuxt2 })
```

Comment:
```suggestion
```

Reply:
I actually left it intentional but no strong opinion to keep it either.